MINT.postman_collection.json

```diff
@@ -144,7 +145,7 @@
 						"body": {
 							"mode": "graphql",
 							"graphql": {
-								"query": "query modelPlanCollection {\n    modelPlanCollection(filter: WITH_CR_TDLS) {\n        id\n        modelName\n        abbreviation\n        archived\n        createdBy\n        createdDts\n        modifiedBy\n        modifiedDts\n        status\n        isFavorite\n        isCollaborator\n        lastModifiedDts\n    }\n}",
```

Comment:
lastModifiedDts should be opSolutionLastModifiedDts, I corrected this query since I was modifying it anyway according to boyscout rules